packages/editor-ui/src/components/ExternalSecretsProviderCard.ee.vue

```diff
@@ -54,7 +54,7 @@ const canConnect = computed(() => {
 });
 
 const formattedDate = computed((provider: ExternalSecretsProvider) => {
-	return DateTime.fromISO(props.provider.connectedAt).toFormat('dd LLL yyyy');
+	return DateTime.fromISO(props.provider.connectedAt ?? new Date()).toFormat('dd LLL yyyy');
```

Comment:
Fix for pre-existing bug where `props.provider.connectedAt` was `null` immediately on connect, causing `Invalid DateTime` to display.